tests/testPhoSimCatalogs.py

```diff
@@ -104,6 +161,226 @@ def testCatalog(self):
         if os.path.exists(catName):
             os.unlink(catName)
 
+    def testHeaderMap(self):
+        """
+        Test the behavior of the phoSimHeaderMap
+        """
+        testBulge = PhoSimCatalogSersic2D(self.bulgeDB, obs_metadata=self.obs_metadata)
+        testBulge.phoSimHeaderMap = {'lunar_distance': ('dist2moon', None),
+                                     'rotation_of_the_telescope': ('rottelpos', np.degrees)}
```

Comment:
Since the default map contians a lambda function, I think we should add a test on an example of that as well.